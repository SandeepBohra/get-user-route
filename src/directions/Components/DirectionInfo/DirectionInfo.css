.DirectionInfo {
    margin-top: 50px;
    font-size: 16px;
    font-weight: 500;
}