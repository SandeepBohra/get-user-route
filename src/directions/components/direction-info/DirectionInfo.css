.direction-info {
    margin-top: 50px;
    font-size: 16px;
    font-weight: 500;
}