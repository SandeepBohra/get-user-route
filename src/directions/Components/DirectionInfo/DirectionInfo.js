import React,  { Component } from 'react';
import './DirectionInfo.css'

export const DirectionInfo = (props) => (
    <div className="DirectionInfo">
        <div className="route-distance">
            <label>{`Total distance: ${props.totalDistance}`}</label>
        </div>
        <div className="route-time">
            <label>{`Total time: ${props.totalTime}`}</label>
        </div>
    </div>
)