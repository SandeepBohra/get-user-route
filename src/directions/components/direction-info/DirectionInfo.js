import React from 'react';
import './DirectionInfo.css'

class DirectionInfo extends React.PureComponent {
    render() {
        return (
            <div className="direction-info">
                <div className="route-distance">
                    <label>{`Total distance: ${this.props.totalDistance}`}</label>
                </div>
                <div className="route-time">
                    <label>{`Total time: ${this.props.totalTime}`}</label>
                </div>
            </div>
        )
    }
}


export default DirectionInfo;