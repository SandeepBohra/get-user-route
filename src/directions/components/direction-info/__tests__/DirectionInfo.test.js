import React from 'react';
import renderer from 'react-test-renderer';
import DirectionInfo from '../DirectionInfo';
import { shallow } from 'enzyme';

describe('<DirectionInfo /> component', () => {
  
  it('should match the snapshot', () => {
    const output = renderer.create(<DirectionInfo />).toJSON();
    expect(output).toMatchSnapshot();
  })

  it('Should render coorectly', () => {
      const container = shallow(< DirectionInfo />);
      
      expect(container.find('div').length).toBe(3);
  })
})