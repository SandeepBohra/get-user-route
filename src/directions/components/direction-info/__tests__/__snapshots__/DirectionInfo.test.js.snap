// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DirectionInfo /> component should match the snapshot 1`] = `
<div
  className="direction-info"
>
  <div
    className="route-distance"
  >
    <label>
      Total distance: undefined
    </label>
  </div>
  <div
    className="route-time"
  >
    <label>
      Total time: undefined
    </label>
  </div>
</div>
`;
